message/message.go

```diff
@@ -44,6 +53,8 @@ type Message struct {
 	//
 	// Control messages trigger special behavior in transforms and conditions.
 	ctrl bool
+
+	flags []any
```

Comment:
Is flags every a non `int` value from the `iota` above? If not, maybe this should have a stronger type than any.

Reply:
fixed